lib/data/multipleBackendGateway.js

```diff
@@ -62,7 +63,28 @@ const multipleBackendGateway = {
         return client.delete(objectGetInfo, reqUids, callback);
     },
 
-    // checkHealth:
+    healthcheck: (log, callback) => {
+        const multBackendResp = {};
+        async.each(Object.keys(clients), (location, cb) => {
+            const client = clients[location];
+            if (client.clientType && client.clientType === 'scality') {
```

Comment:
Thanks. It's minor, but an undefined `client.clientType` will also not equal `'scality'`. That's what I meant 😄 .